gallery/v2_transforms/plot_transforms_v2.py

```diff
@@ -7,103 +7,253 @@
     Try on `collab <https://colab.research.google.com/github/pytorch/vision/blob/gh-pages/main/_generated_ipynb_notebooks/plot_transforms_v2.ipynb>`_
     or :ref:`go to the end <sphx_glr_download_auto_examples_v2_transforms_plot_transforms_v2.py>` to download the full example code.
 
-Most computer vision tasks are not supported out of the box by ``torchvision.transforms`` v1, since it only supports
-images. ``torchvision.transforms.v2`` enables jointly transforming images, videos, bounding boxes, and masks. This
-example showcases the core functionality of the new ``torchvision.transforms.v2`` API.
+This example illustrates all of what you need to know to get started with the
+new :mod:`torchvision.transforms.v2` API. We'll cover simple tasks like
+image classification, and more advanced ones like object detection /
+segmentation.
 """
 
-import pathlib
-
+# %%
+# First, a bit of setup
+from pathlib import Path
 import torch
+import matplotlib.pyplot as plt
+plt.rcParams["savefig.bbox"] = 'tight'
 
+from torchvision.transforms import v2
+from torchvision.io import read_image
 
-def load_data():
-    from torchvision.io import read_image
-    from torchvision import datapoints
-    from torchvision.ops import masks_to_boxes
+torch.manual_seed(1)
 
-    assets_directory = pathlib.Path("../assets")
+# If you're trying to run that on collab, you can download the assets and the
+# helpers from https://github.com/pytorch/vision/tree/main/gallery/
+from helpers import plot
+img = read_image(str(Path('../assets') / 'astronaut.jpg'))
```

Comment:
This will also not work on Collab without user interference, right?

Reply:
probably not